src/DynamoCrypto/DynamoCrypto.csproj

```diff
@@ -9,12 +9,7 @@
     <RootNamespace>DynamoCrypto</RootNamespace>
     <AssemblyName>DynamoCrypto</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>..\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
+    <ItemGroup>
```

Comment:
reminds me  - we should get rid of this project and the whole updater infrastructure when we can...